### ISSUE TITLE
Using Works on ARM resource

### ISSUE THREAD

**Issue by junaruga** [team-gogut/yamp-arm#5]
As we open sourced the code, it's time to use ARM CI/CD environment "Works on ARM" that is maybe a server resource rather than just CI like Shippable CI.
https://www.worksonarm.com/cluster/

It is possible to create an issue on below repository.
https://github.com/worksonarm/cluster/

We can choose the plan from
https://www.packet.com/bare-metal/

**Comment by vielmetti** [comment]
I'd welcome a proposal to @worksonarm !

**Comment by junaruga** [comment]
@vielmetti thanks for your offer!

I positively think of using WorksOnArm!
Right now I am trying to create an aarch64 container image on aarch64 server to register it to a container registry.
But using Shippable CI, the building time exceeds the maximum time: 60 minutes.

WorksOnArm provides a server function by SSH login right?

**Comment by vielmetti** [comment]
Yes, the Works on Arm service has access to bare metal hardware, from which you can set up your own development and build environment with full access including server login via SSH.

You might also look into using Drone Cloud (https://cloud.drone.io) which is similarly hosted CI at Packet, I think their build environment is pretty good too.

**Comment by junaruga** [comment]
@vielmetti thank you for the information. Now we are requesting the server resource!
https://github.com/WorksOnArm/cluster/issues/158
